app/models/concerns/notifiable.rb

```diff
@@ -8,13 +8,24 @@ module Notifiable
   def create_notifications(action:, recipients:)
     ActiveRecord::Base.transaction do
       recipients.each do |recipient|
-        Notification.create(
+        notification = Notification.create(
           action: action,
           actor: user,
           notifiable: self,
           recipient: recipient
         )
+
+        broadcast_to_user(notification, recipient)
       end
     end
   end
+
+  def broadcast_to_user(notification, user)
+    project = self.commentable.project
+    notification_html = NotificationsController.render(
```

Comment:
could this `render` call be simplified? `render(notification)` ?
May be we can get the project from the notifications itself, not need to pass `notification_project`

Reply:
More importantly, why are we calling controller methods, rendering HTML and broadcasting to websockets from within a *model*? These are clearly things that belong in the controller layer.

Reply:
I don't think this is that terrible, but option would be to not render anything and with js force a click to trigger the dropdown reload via ajax